src/ducks/projects.js

```diff
@@ -32,7 +32,8 @@ const fetchProjects = () => {
       type: FETCH_PROJECTS,
     });
     const query = {
-      current_user_roles: ['owner', 'translator']
+      current_user_roles: ['owner', 'translator'],
+      include: ['workflows']
```

Comment:
Can we filter workflows? I can't think of a way involving not fetching all projects.